### PR TITLE
Clean up some remants of the past

### PR DESCRIPTION
**Pull request by davidfowl**
- Remove Application from HttpConnectionContext and Http1ConnectionContext (it's no longer required)

### CODE REVIEW

File: src/Kestrel.Core/Internal/Http2/Http2FrameWriter.cs [hunk 1]
```diff
@@ -66,11 +66,20 @@ public void Complete()
             }
         }
 
-        public void Abort(ConnectionAbortedException ex)
+        public void Abort(ConnectionAbortedException error)
         {
-            // TODO: Really abort the connection using the ConnectionContex like Http1OutputProducer.
```
Comment (halter73) [inline src/Kestrel.Core/Internal/Http2/Http2FrameWriter.cs]: Nice!

**Comment by davidfowl** [comment]
Oh wtf, conflicts..

**Comment by davidfowl** [comment]
@dotnet-bot test this

**Comment by davidfowl** [comment]
@dotnet-bot help

**Comment by davidfowl** [comment]
@dotnet-bot test

**Comment by davidfowl** [comment]
@dotnet-bot test Windows Release x64 Build

**Comment by davidfowl** [comment]
@dotnet-bot test Windows Debug x64 Build

**Comment by davidfowl** [comment]
@dotnet-bot test Ubuntu 16.04 Release Build

**Comment by davidfowl** [comment]
@dotnet-bot test OSX 10.12 Debug Build

**Comment by halter73** [comment]
So you're merging this into master, right? :wink:

**Comment by davidfowl** [comment]
Isn't that automated 😉

**Pull request merged**